examples/thalamus_sample_dir/config/features/units.json

```diff
@@ -0,0 +1,44 @@
+{
+    "inv_time_to_first_spike": "Hz",
+    "inv_first_ISI": "Hz",
+    "inv_second_ISI": "Hz",
+    "inv_third_ISI": "Hz",
+    "inv_fourth_ISI": "Hz",
+    "inv_fifth_ISI": "Hz",
+    "inv_last_ISI": "Hz",
+    "time_to_last_spike": "ms",
+    "Spikecount": "constant",
+    "Spikecount_stimint": "constant",
+    "number_initial_spikes": "constant",
+    "ISI_semilog_slope": "ms",
+    "ISI_log_slope": "ms",
+    "check_AISInitiation": "ms",
```

Comment:
This should be 'constant', not 'ms'. The units are also false in efel's documentation, I'll change that in my next efel MR.

Could you also change it in the units.json file of sscx example, please?

Reply:
Alright. Only the `check_AISInitiation` should be "constant", right?

Reply:
Yes, only `check_AISInitiation` into "constant"

Reply:
Alright, updated in 78e5fd5. The file was created by e-model-packages. That's also updated in an MR